VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -532,10 +532,18 @@ def do_status_report(self, operation, status, status_code, message, taskId = Non
         sub_stat = []
         stat_rept = []
         self.add_telemetry_data()
+        snapshotCreatorInfo = ""
+
```

Comment:
we also need statusUploadError like in windows (that tells whether VM did not have network)